packages/plugins/src/plugins/morpho/constants.ts

```diff
@@ -18,3 +20,11 @@ export const Underlying = {
 
 export const wethAddress = '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2';
 export const wethDecimals = 18;
+
+export const morphoApiUrl = 'https://blue-api.morpho.org/graphql';
+export const morphoMarketsCachePrefix = `${platformId}-markets`;
+export const morphoVaultsCachePrefix = `${platformId}-vaults`;
+
+export const networkIdToMorphoContract = {
+  [NetworkId.ethereum]: '0xbbbbbbbbbb9cc5e90e3b3af64bdaf62c37eeffcb',
+} as Record<EvmNetworkIdType, string>;
```

Comment:
We should use stricter typing:
```suggestion
} as Record<EvmNetworkIdType, Address>;
```
(import from `miev`).